cluster/addons/metrics-server/README.md

```diff
@@ -0,0 +1,20 @@
+# Metrics Server
+
+[Metrics Server](https://github.com/kubernetes-incubator/metrics-server) exposes
+core Kubernetes metrics via metrics API.
+
+More details can be found in [Core metrics pipeline documentation](https://kubernetes.io/docs/tasks/debug-application-cluster/core-metrics-pipeline/).
+
+## Troubleshooting
+
+Metrics Server supports up to 30 pods per cluster node. In clusters where there are more running pods, Metrics Server may be throttled or fail with OOM error. Starting with Kubernetes 1.9.2, Metrics Server resource requirements may be overwritten manually. [Learn more about Addon Resizer configuration](https://github.com/kubernetes/autoscaler/tree/master/addon-resizer#addon-resizer-configuration)
+
+### Important notices
+
+1. Decreasing resource requirements for cluster addons may cause system instability. The effects may include (but are not limited to):
+  - Metrics not being exported
+  - Horizontal Pod Autoscaler not working
+  - `kubectl top` not working
+1. Overwritten configuration persists through cluster updates, therefore may cause all effects above after a cluster update.
```

Comment:
ditto

Reply:
All future versions may be affected.

Reply:
Future starting from which version exactly? Please put that version in the doc.

Reply:
I'm not sure what number do you want me to put there. Addons can be configured starting with 1.9.2, which is specified above. This sentence refers to all updates after the addon configuration is modified, so technically > 1.9.2. Do you want me to repeat this number here? I would prefer to leave it as it is, ALL clusters updates can cause above effects if the configuration is modified, which is exactly what this sentence says.